dev/index.js

```diff
@@ -83,5 +83,8 @@ renderSiteApp(
             }
         }
     ],
-    "#/folder-1/file-1"
+    {
+        gettingStartedLink: "#/folder-1/file-1",
```

Comment:
Can you update the docs here: https://github.com/cerner/carbon-site-helpers#usage